Project.toml

```diff
@@ -5,7 +5,6 @@ version = "0.16.3"
 
```

Comment:
Reminder to bump major version because of breaking changes in API. Version 1.0.0 incoming! 🎉